.prettierignore

```diff
@@ -6,6 +6,8 @@ tmp/*
 .DS_Store
 public/doc/*
 db/*.sql
+db/schema.rb
+db/views_schema.rb
 db/views_schema.old
```

Comment:
Prettier touching the db/schema is just annoying. It explains the massive schema diff here. Did try and get that in a two part commit, but it didn't work out as intended and somehow ended up as one. Sorry about that.